Fix smtpPort default from 25 to undefined

daichi703n opened the pull request:
## Summary
Fix the default value of **SMTP Port** in the Email alert setting from 25 to ~~null~~ **undefined**.

## Why
The default value for `smtpPort` should not be set, as it may unexpectedly overwrite the base configuration of `smtp_port` and break existing alert operation.

Common settings such as `smtp_host` and `smtp_port` should be configured in `config.yaml` or `BaseRule.config`. Even if `smtp_port` is not set, the default value of 25 will be applied by ElastAlert, so there is no need to specify it explicitly in each alert rule.

nsano-rururu commented:
In that case, the following code setting is also wrong.

https://github.com/johnsusek/praeco/blob/master/src/store/config/alert.js#L147
https://github.com/johnsusek/praeco/blob/master/src/store/config/index.js#L368

nsano-rururu commented:
Now, you can set it on the screen.

nsano-rururu commented:
I understand that this is due to the following implementation part of elastalert2:
https://github.com/jertel/elastalert2/blob/master/elastalert/alerters/email.py#L107
![image](https://github.com/user-attachments/assets/ace8040c-2a6d-43fd-abb4-940939356f8e)

nsano-rururu commented:
In that case, the following line in the test code should not be output to yaml when smtp_port is not specified, so it should be possible to delete it.
https://github.com/johnsusek/praeco/blob/master/tests/unit/specs/alert/ConfigYamlEmail.spec.js#L45

daichi703n commented:
Thank you for your review. I have removed the parts you pointed out.

Upon conducting additional tests, I found that using `null` sets the initial value of the form to `0`. As a fix, I have changed it to `undefined`.

nsano-rururu commented:
The port numbers seem to range from 0 to 65535, so you may want to add minimum and maximum values.

https://www.nic.ad.jp/ja/basics/terms/port-number.html#:~:text=%E3%83%9D%E3%83%BC%E3%83%88%E7%95%AA%E5%8F%B7%E3%81%AF16%E3%83%93%E3%83%83%E3%83%88,%EF%BD%9E65535%E7%95%AA%E3%81%BE%E3%81%A7%E3%81%82%E3%82%8A%E3%81%BE%E3%81%99%E3%80%82

https://github.com/johnsusek/praeco/blob/master/src/components/config/alert/ConfigAlertEmail.vue#L62

before
```vue
<el-input-number id="smtpPort" :value="smtpPort" :disabled="viewOnly" @input="smtpPort = $event" />
```
aftter

```vue
<el-input-number id="smtpPort" :value="smtpPort" :disabled="viewOnly" :min="0" :max="65535" @input="smtpPort = $event" />
```

nsano-rururu commented:
@daichi703n 
I haven't checked it with elastalert2, so I'll discard the previous post.
I'll merge it today.
